ft-sdk/src/layout.rs

```diff
@@ -24,7 +24,7 @@ where
     E: std::fmt::Debug + From<ft_sdk::Error>,
 {
     // Validate content in Action struct
-    fn validate(c: &mut L) -> Result<Self, E>
+    fn validate(c: &mut L, start_time: chrono::DateTime<chrono::Utc>) -> Result<Self, E>
```

Comment:
Nopes, time is already part of In, which is pass to from_in, if you care about time, save In.